components/Alert/components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
```

Comment:
We should need only one storybook file ideally. See if you can merge the previous one with this.